core/templates/pages/blog-dashboard-page/blog-post-editor/blog-post-editor.component.spec.ts

```diff
@@ -323,6 +321,7 @@ describe('Blog Post Editor Component', () => {
 
     component.saveDraft();
 
+    expect(component.saveInProgress).toBe(true);
```

Comment:
Nit:

```suggestion
    expect(component.saveInProgress).toBeTrue();
```

ditto elsewhere

Reply:
Done